trace/validation_trace.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/graph-gophers/graphql-go/errors"
 )
 
-type TraceValidationFinishFunc = TraceQueryFinishFunc
+type TraceValidationFinishFunc TraceQueryFinishFunc
```

Comment:
TBH, `TraceValidationFinishFunc` is only ever used in this file. IMHO, we don't even need type aliasing here. I'd be happy to merge your PR but I think that the approach is wrong. Instead, we need to remove this line altogether and replace `TraceValidationFinishFunc` on lines 10 and 15 with `TraceQueryFinishFunc`.

Reply:
WDYT?